tests/DummyData.spec.js

```diff
@@ -9,155 +9,144 @@ beforeAll(async () => {
     Uint8Array.from([ 0x03, 0x08, 0x44, 0x88, 0xd9, 0xbd, 0x82, 0xf8, 0x7b, 0x88, 0x69, 0xc4, 0xde, 0x03, 0x19, 0xe2, 0xc3, 0x95, 0xd3, 0x7d, 0x57, 0x17, 0x5a, 0x34, 0x48, 0xd2, 0x4c, 0x0d, 0xa5, 0x3d, 0xe8, 0xb4, 0x1a, 0x41, 0x13, 0xa2, 0xdc, 0xae, 0x54, 0x03, 0x0d, 0xce, 0xaf, 0x2a, 0x59, 0x10, 0xf1, 0x71, 0x7b, 0x0b, 0x2a, 0x90, 0x05, 0x6f, 0xb3, 0x46 ]),
     // eslint-disable-next-line max-len
     Uint8Array.from([ 0x03, 0x08, 0x38, 0x8f, 0x65, 0x9f, 0x23, 0xfb, 0x80, 0x13, 0xd5, 0xef, 0x86, 0xdc, 0x4d, 0xdc, 0xd7, 0x13, 0xf4, 0x00, 0x34, 0x90, 0xe4, 0xc7, 0x9e, 0x78, 0x84, 0xa5, 0x5e, 0x21, 0x3a, 0xdb, 0x6a, 0xe1, 0x94, 0x48, 0xbd, 0x93, 0xed, 0xd1, 0x2b, 0xfd, 0x0e, 0x1f, 0x34, 0xd5, 0x4b, 0x38, 0x17, 0x4b, 0x8b, 0xf2, 0xd7, 0x4e, 0x1c, 0x10 ]),
 ];
 
-/** @type {string} */
-Dummy.encryptionPassword = 'password';
+const hashes = () => secrets.map(secret => {
+    const input = secret instanceof Nimiq.Entropy
+        ? secret.serialize()
+        : Nimiq.PublicKey.derive(secret).toAddress().serialize();
+
+    return Nimiq.BufferUtils.toHex(Nimiq.Hash.blake2b(input).subarray(0, 32))
+});
 
-/** @type {KeyInfo[]} */
-Dummy.keyInfos = [
+const encryptionPassword = 'password';
+const encryptionPassword2 = 'password2';
+
+const keyInfos = [
     new KeyInfo(
-        '2ec615522906',
+        1,
         Nimiq.Secret.Type.PRIVATE_KEY,
         true,
         false,
     ),
     new KeyInfo(
-        'ef553f34a779',
+        2,
         Nimiq.Secret.Type.ENTROPY,
         false,
         false,
     ),
 ];
 
-/** @type {KeyInfo[]} */
-Dummy.cookieKeyInfos = [
-    new KeyInfo(
-        '2ec615522906',
-        Nimiq.Secret.Type.PRIVATE_KEY,
-        true,
-        false,
-    ),
-    new KeyInfo(
-        'ef553f34a779',
-        Nimiq.Secret.Type.ENTROPY,
-        true,
-        false,
-    ),
-];
+const cookieKeyInfos = keyInfos.map(/** @param {KeyInfo} x */x => new KeyInfo(x.id, x.type, true, x.hasPin));
 
-/** @type {KeyguardRequest.KeyInfoObject[]} */
-Dummy.keyInfoObjects = [
-    {
-        id: Dummy.keyInfos[0].id,
-        type: Dummy.keyInfos[0].type,
-        hasPin: Dummy.keyInfos[0].hasPin,
-    },
-    {
-        id: Dummy.keyInfos[1].id,
-        type: Dummy.keyInfos[1].type,
-        hasPin: Dummy.keyInfos[1].hasPin,
-    },
-];
+const keyInfoObjects = keyInfos.map(/** @param {KeyInfo} x */x => 
+    ({
+        id: x.id,
+        type: x.type,
+        hasPin: x.hasPin
+    }));
 
 const _purposeIdBuf = new Nimiq.SerialBuffer(4);
 _purposeIdBuf.writeUint32(Nimiq.Entropy.PURPOSE_ID);
 const _purposeIdArray = Array.from(_purposeIdBuf.subarray(0, 4));
 
-/** @type {KeyRecord[]} */
-Dummy.keyRecords = [
-    Object.assign({}, Dummy.keyInfoObjects[0], { secret: Dummy.encryptedKeys[0] }),
-    Object.assign({}, Dummy.keyInfoObjects[1], { secret: new Uint8Array(_purposeIdArray.concat(Array.from(Dummy.keys[1]))) }),
+const keyRecords = () => [
+    Object.assign({}, {
+        type: keyInfoObjects[0].type,
```

Comment:
Why use `assign` if not inheriting from anything? The copied properties are still references, when they are objects, no?